TicTacToeApp/BoardButtons.swift

```diff
@@ -0,0 +1,36 @@
+
+import UIKit
+
+public class BoardButtons {
```

Comment:
I like this abstraction. It's a good abstraction on it's own and seems to be a good stepping stone towards a custom UI element if/when that is desired.